index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <p>The man who saved a billion lives</p>
+      <div id="img-div">
+        <img
+          src="/images/tribute-page-main-image.jpeg"
+          alt="tribute image"
+          id="image"
+        />
```

Comment:
@glorymugizi Good job so far. Kindly verify the source path. It appears to be broken because the tribute-page-main-image does not appear.
👇
![image](https://user-images.githubusercontent.com/75784269/205673467-010118e2-1c71-4ec3-8dfd-410a8e6a0a5d.png)
Thank you.